src/openrct2/scripting/bindings/ride/ScTrackSegment.h

```diff
@@ -19,6 +19,18 @@
 
 namespace OpenRCT2::Scripting
 {
+    template<> inline DukValue ToDuk(duk_context* ctx, const rct_vehicle_info& value)
+    {
+        DukObject dukSubposition(ctx);
+        dukSubposition.Set("x", value.x);
+        dukSubposition.Set("y", value.y);
+        dukSubposition.Set("z", value.z);
+        dukSubposition.Set("direction", value.direction);
```

Comment:
I would prefer it expose it as yaw, pitch and roll as that is the names ideally the codebase should be using.

Reply:
I can rename them but, as mentioned in the main post, it would add inconsistency with the already existing external plugin API use of 'angle', 'slope' and 'banking' [here](https://github.com/OpenRCT2/OpenRCT2/blob/7f29e4e39c76dbf7039162368ec8bb604e943d03/distribution/openrct2.d.ts#L1194-L1202). Is that inconsistency preferred over the current one?

Reply:
those are for tracks though its a different incomparable property this is for vehicles.

Reply:
In the end they're the same slope/banking/pitching values. :P But alright, I'll rename them.